linkerd/proxy/http/src/glue.rs

```diff
@@ -163,15 +163,15 @@ impl<C, T> HyperConnect<C, T> {
     pub(super) fn new(connect: C, target: T, absolute_form: bool) -> Self {
         HyperConnect {
             connect,
-            absolute_form,
             target,
+            absolute_form,
         }
     }
 }
 
 impl<C, T> Service<hyper::Uri> for HyperConnect<C, T>
 where
-    C: MakeConnection<T> + Clone + Send + Sync,
+    C: MakeConnection<(crate::Version, T)> + Clone + Send + Sync,
```

Comment:
this kind of makes me wonder if the `Connect<T>` type should just live in `linkerd-proxy-http`, but i guess then we wouldn't be able to write all the necessary `Param` impls for `Connect<T>`...it's a shame we can't just have a 

```rust
impl<T, P> Param<P> for Connect<T> where T: Param<P> { ... }
```
without conflicting with the existing `Param` blanket impls :(

Reply:
yeah, agreed that it would be a lot nicer with blanket impulse. in general, though, I prefer to keep the target types in the modules from which they are emitted.